python/xoscar/collective/gloo/test/__init__.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2022-2023 XProbe Inc.
```

Comment:
The dir that contains UT should be named to ``tests``. Please also modify the ``test`` dir in ``python/xoscar/collective/rendezvous``

Reply:
Done